tests/sles4sap/trento/cluster_deploy.pm

```diff
@@ -10,22 +10,22 @@ use Mojo::Base 'publiccloud::basetest';
 use testapi;
 use serial_terminal 'select_serial_terminal';
 use qesapdeployment 'qesap_upload_logs';
-use base 'trento';
+use trento qw(deploy_qesap destroy_qesap get_resource_group get_qesap_resource_group get_vnet az_delete_group k8s_logs trento_support);
```

Comment:
Since there are so many methods being imported, perhaps `use trento;` could be better. But it's a minor thing.